app/components/authorization-code/login-custom/login-custom.controller.js

```diff
@@ -27,6 +27,11 @@ class LoginCustomController {
         responseType: 'code',
         scopes: ['openid', 'email', 'profile'],
       },
+      i18n: {
+        en: {
+          'primaryauth.title': 'Use john/Asdf1234 for the mock Okta server',
```

Comment:
Since these are sample apps, not adding in i18n (i.e. we wouldn't get these strings translated)